MiniTeks/mks-server
# Add mksTask Controller

#### This PR contains a controller to listen for add, update and deletion of resources, and invokes tekton apis when a new mksTask is added to create tekton Task. 

* pkg now  has a reconciler/ having a controller.
* config folder contains three yaml files to test the controller.

##### To test:- 
1. Apply the mksTask CRD - `k apply -f config/mksTaskCRD.yaml`
2. Build and execute the program. (see README.md)
